src/views/Bitcoin.vue

```diff
@@ -202,8 +202,14 @@
                 <span class="align-self-end">Synchronized</span>
                 <h3 class="font-weight-normal mb-0">
                   <span v-if="syncPercent !== -1">
-                    {{ syncPercent >= 99.99 ? 100 : syncPercent }}
-                    <small class>%</small>
+                    <CountUp
+                      :value="{
+                        endVal: syncPercent >= 99.99 ? 100 : syncPercent,
+                        decimalPlaces: syncPercent >= 99.99 ? 0 : 2
+                      }"
+                      suffix="%"
+                      v-if="syncPercent !== -1"
```

Comment:
We can remove this duplicate check as it's already added on the parent `span` element:

```suggestion

```

Reply:
Good catch! I have updated the PR @mayankchhabra